src/components/ArticleCard.jsx

```diff
@@ -7,7 +7,9 @@ export default function ArticleCard({ article }) {
             <img src={article.article_img_url}/>
         </Link>
         <div className="ArticleCard-text">
-          <h4 className="article-topic">{article.topic}</h4>
+          <Link to={`/articles/?topic=${article.topic}`}>
+            <h4 className="article-topic">{article.topic}</h4>
```

Comment:
Avoid header stepping. Do we have other headers before this e.g. h2 and h3? Does this even need to be a header?